packages/react/src/components/form/Checkbox/Group/Group.tsx

```diff
@@ -0,0 +1,80 @@
+import type { ReactNode } from 'react';
+import React, { useState, forwardRef, createContext } from 'react';
+
+import type { FieldsetProps } from '../../Fieldset';
+import { Fieldset } from '../../Fieldset';
+
+import classes from './Group.module.css';
+
+export type CheckboxGroupContextProps = {
+  value?: string[];
+  defaultValue?: string[];
+  toggleValue: (value: string) => void;
+};
+
+export const CheckboxGroupContext =
+  createContext<CheckboxGroupContextProps | null>(null);
+
+export type CheckboxGroupProps = {
+  /** Collection of `Checkbox` components */
+  children?: ReactNode;
+  /** Controlled state for  `Checkbox`'s */
+  value?: string[];
+  /** Default checked `Checkbox`'s */
+  defaultValue?: string[];
+  /** Callback event with checked `Checkbox` values */
+  onChange?: (value: string[]) => void;
+} & Omit<FieldsetProps, 'onChange'>;
+
+export const CheckboxGroup = forwardRef<
+  HTMLFieldSetElement,
+  CheckboxGroupProps
+>(
+  (
+    {
+      onChange,
+      children,
+      value,
+      readOnly,
+      defaultValue,
+      size = 'medium',
+      ...rest
+    },
+    ref,
+  ) => {
+    const [internalValue, setInternalValue] = useState<string[]>(
+      defaultValue ?? [],
+    );
+
+    const toggleValue: CheckboxGroupContextProps['toggleValue'] = (
+      checkboxValue,
+    ) => {
+      const currentValue = value ?? internalValue;
+      const updatedValue = currentValue.includes(checkboxValue)
+        ? currentValue.filter((x) => x !== checkboxValue)
+        : [...currentValue, checkboxValue];
+
+      value ?? setInternalValue(updatedValue);
```

Comment:
Good catch!

Er riktig det: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

> The nullish coalescing (??) operator is a logical operator that returns its right-hand side operand when its left-hand side operand is [null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/null) or [undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined), and otherwise returns its left-hand side operand.

Tenkte jeg kunne være smart å bruke denne operatoren for å sjekke på både undefined og null, men det er vel egentlig feil bruk av den operatoren siden den returner venstre siden dersom den er defined. Viktigste her å sjekke om value er undefined egentlig.

Fikser det!